src/Playwright/Core/APIRequestContext.cs

```diff
@@ -164,6 +159,28 @@ public async Task<IAPIResponse> FetchAsync(string url, APIRequestContextOptions
         return new APIResponse(this, response?.GetProperty("response").ToObject<Transport.Protocol.APIResponse>());
     }
 
+    private static IEnumerable<NameValue> QueryParamsToProtocol(IEnumerable<KeyValuePair<string, object>> @params, string paramsString)
+    {
+        if (@params != null)
```

Comment:
Why `@params` and not `params`?

Reply:
Reserved keyword I guess: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/method-parameters#params-modifier